server/sonar-web/src/main/js/apps/organizations/components/forms/AddMemberForm.js

```diff
@@ -72,6 +72,12 @@ export default class AddMemberForm extends React.PureComponent {
     }
   };
 
+  handleClose = (e /*: MouseEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)